roles/polarion/tasks/main.yml

```diff
@@ -108,10 +123,11 @@
           source "{{ cifmw_polarion_jump_repo_dir }}/jump-venv/bin/activate" &&
           {{ cifmw_polarion_jump_repo_dir }}/jump-venv/bin/python jump.py
           --testrun-id={{ cifmw_polarion_testrun_id }}
```

Comment:
The change will **probably** not work as intended when `cifmw_polarion_testrun_id` is specified (no `create-new-test-run`). Update of the README.md might be a good idea if it is really true. It might surprise somebody later.

Reply:
I'll update the README to reflect this behavior